python-connectors/github_search-pull-requests/connector.json

```diff
@@ -0,0 +1,95 @@
+{
+    "meta" : {
+        "label": "Search GitHub Pull Requests",
+        "description": "Allows to search Pull Requests",
+        "icon": "icon-github-sign"
+    },
+    "readable": true,
+    "writable": false,
+
+    "params": [
+        {
+            "name": "personal_access_token_credentials_preset",
+            "label": "Github Personal Access Token",
+            "description": "For more information on how to create your token: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token",
+            "type":"PRESET",
+            "parameterSetId": "personal-access-token"
+        },
+        {
+            "name": "owner",
+            "label": "Git owner",
+            "description": "Owner to fetch Pull Requests from (as in the full name {owner}/{repo}).",
+            "type":"STRING"
+        },
+        {
+            "name": "github_team_handles",
+            "type": "STRINGS",
+            "label": "Github team usernames",
+            "description": "Entered value can also be a single variable containing a list of user names (${var_name})",
+            "mandatory": true,
+            "canSelectForeign": false,
+            "markCreatedAsBuilt": false,
+            "allowDuplicates": true,
+            "getChoicesFromPython": false,
+            "triggerParameters": [],
+            "disableAutoReload": false,
+            "canCreateDataset": false
+        },
+		{
+			"name" : "link_to_users",
+            "label" : "Select the link to the users",
+			"type" : "SELECT",
+            "selectChoices": [
+                { "value": "open_by", "label": "Open by"},
+                { "value": "reviewed_by", "label": "Reviewed by"},
+                { "value": "all", "label": "All"}
+            ],
+            "defaultValue": "open_by"
+		},
+		{
+			"name" : "state",
```

Comment:
Out of the box from PyCharm those files are properly indented. I can reindent all to make sure there are no more bad indentation but it will change the default indendation of the file so not sure it's better 🤷‍♂️